mission-8/src/services/notifications.service.ts

```diff
@@ -0,0 +1,72 @@
+import { NotificationsRepository } from '../repositories/notifications.repository.js';
+import { NotificationType } from '@prisma/client';
+
+export class NotificationsService {
+  private static instance: NotificationsService;
+  notificationsRepository = new NotificationsRepository();
+
+  // Socket.IO instance will be injected later
+  private io: any;
```

Comment:
```suggestion
  private io: Server;
```

`import { Server } from 'socket.io';`
타입 안정성을 위해 변경해주시면 좋습니다.